Haskell/Models/Score.txt

```diff
@@ -0,0 +1 @@
+12
```

Comment:
This file should be added to `.gitignore`, as it is not meant to be committed, right?

Reply:
Also the `Score.txt` file should be outside of the Models folder